bitnami/concourse/values.yaml

```diff
@@ -109,6 +109,17 @@ secrets:
   ## https://concourse-ci.org/global-resources.html#complications-with-reusing-containers
   ##
   teamAuthorizedKeys: []
+  ## Secrets for Conjur credentials manager.
+  ## @param secrets.conjurAccount Account for Conjur auth provider.
+  conjurAccount: ""
+  ## @param secrets.conjurAuthnLogin Host username for Conjur auth provider.
+  conjurAuthnLogin: ""
+  ## @param secrets.conjurAuthnApiKey API key for host used for Conjur auth provider. Either API key or token file can be used, but not both.
+  conjurAuthnApiKey: ""
```

Comment:
If this is mandatory to set when the token file is not set, we may need to add a validation to ensure it is set

Reply:
I added some validations in NOTES.txt.